src/assets/usage-examples-code/sprites/sprite_set_y/sprite_set_y-9-sprite-y.txt

```diff
@@ -0,0 +1,3 @@
+### sprite_set_y() usage example
+
+The following code is an example of using sprite set y which sets the y position of a sprite on a splashkit window.
```

Comment:
Update this to be a bit clearer and use capital first letters on the function name with a proper link.

Reply:
I have done as asked.